scripts/pi-hole/js/queryads.js

```diff
@@ -73,12 +73,12 @@ function eventsource() {
 
     // IE does not support EventSource - load whole content at once
     if (typeof EventSource !== "function") {
-        httpGet(ta,quiet,"/admin/scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+exact+"&IE");
+        httpGet(ta,quiet,"scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+exact+"&IE");
         return;
     }
 
     var host = window.location.host;
-    var source = new EventSource("/admin/scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+"&"+exact);
+    var source = new EventSource("scripts/pi-hole/php/queryads.php?domain="+domain.val().toLowerCase()+"&"+exact);
```

Comment:
The `/admin` was put there so that when this script is used on the block page it can reference the correct file. Say that the `/ads/1234.jpg` file was served. Then without `/admin` the script would be sourced from `/ads/scripts/pi-hole/...` which is incorrect.

Reply:
It the usage of this script on a/the block page a WIP? I've looked through the entire project and there are no other pages besides `queryads.php` that load this script.

Reply:
The block page is hosted in the core repo, and uses the script [here](https://github.com/pi-hole/pi-hole/blob/master/advanced/index.php#L144).